### ISSUE TITLE
Documentation for I18n.translate is not supported by YARD (rdoc.info)

### ISSUE THREAD

**Issue by bsingr** [svenfuchs/i18n#185]
I find it quite confusing that the `I18n.translate` (and `I18n.localize`) method documentation ISN'T visible on [rdoc.info](http://rdoc.info/github/svenfuchs/i18n/I18n). However, the `I18n::Backend::Base#translate` method documentation IS visible ([rdoc.info](http://rubydoc.info/github/svenfuchs/i18n/I18n/Backend/Base:translate)). To make things even worse: both methods have different method signatures.

I found a documentation link within the rails repo that uses `I18n.translate` but refers to the documentation for `I18n::Backend::Base#translate` (see rails/rails@c8e057198796f1052f2e3e185f0c777292e75394 and rails/rails#4027).

What I want is a documentation for `I18n.translate` on http://rubydoc.info/github/svenfuchs/i18n/I18n:translate. Sadly this just returns HTTP 404 :(

This is because YARD cannot understand the methods defined within `extend(Mixin.new{...})` introduced in svenfuchs/i18n@c5d9e8cfbda9bdac0dd4f4231d12e5eb8a1c184e.

Running `yard` brings up the following warnings:

```
$ yard
[warn]: in YARD::Handlers::Ruby::ExtendHandler: Undocumentable mixin: YARD::Parser::UndocumentableError for class I18n
[warn]:   in file 'lib/i18n.rb':16:

  16: extend(Module.new {

[warn]: in YARD::Handlers::Ruby::MixinHandler: Undocumentable mixin: YARD::Parser::UndocumentableError for class I18n::ExceptionHandler
[warn]:   in file 'lib/i18n/exceptions.rb':7:

  7: include Module.new {

Files:          44
Modules:        39 (   35 undocumented)
Classes:        22 (   16 undocumented)
Constants:      19 (   18 undocumented)
Methods:       127 (   61 undocumented)
 37.20% documented
```

Does anyone know a solution for that? Probably YARD is already capable of dealing with stuff like that and we just need to set an additional option?

--dpree

**Comment by radar** [comment]
Agreed that it's a yard problem.

**Comment by radar** [comment]
I've filed an issue over there now https://github.com/lsegal/yard/issues/1034. Will leave this issue open as there's likely to be changes in either yard and/or i18n required.

**Comment by radar** [comment]
This is now fixed in master as of bc926ebf98a9ae8f2db843ce16a33a5282d18d35.
